src/app/InteractionModelEngine.h

```diff
@@ -50,6 +50,9 @@
 namespace chip {
 namespace app {
 
+constexpr size_t kMaxSecureSduLength  = 1024;
```

Comment:
Please document that this is the size (of what?) in bytes.

Reply:
sure，will do it in next pr